packages/styletron-react/src/dev-tool.js

```diff
@@ -1,5 +1,7 @@
 /* eslint-env browser */
 /* global module */
+import * as React from "react";
+import * as ReactIs from "react-is";
```

Comment:
This is only in development, but in theory this could still be tree-shaken so I think it's probably worth importing the specific export

```suggestion
import {isForwardRef} from "react-is";
```